.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: 'standard-with-typescript',
```

Comment:
- Add Google eslint config: https://www.npmjs.com/package/eslint-config-google#using-the-google-config-with-eslintrecommended